beets/util/__init__.py

```diff
@@ -216,8 +216,7 @@ def mkdirall(path):
             try:
                 os.mkdir(syspath(ancestor))
             except (OSError, IOError) as exc:
-                raise FilesystemError(exc, 'create', (ancestor,),
-                                      traceback.format_exc())
```

Comment:
Yes, that's a good point—on Python 3, we should just use exception chaining (i.e., [PEP 3134](https://www.python.org/dev/peps/pep-3134/)) with a `from` clause on the `raise` statement. Maybe we can sidestep this issue by using `from` on Python 3 and `format_exc` on Python 2.

I still don't quite understand why that crashes, though! Quite mysterious indeed.